frontend/src/app/general/auth/basic_auth_service/basic_auth_token.service.ts

```diff
@@ -0,0 +1,54 @@
+/*
```

Comment:
Please use the Angular CLI to create new components and services. Generated components never have a `_` in their filename, Angular uses `-` instead.